src/controllers/v1/certificate/index.ts

```diff
@@ -78,6 +78,15 @@ export class CertificateController extends Controller {
     )
   }
 
+  private transformCertNumbers(cert: CertificateRow) {
+    return {
+      ...cert,
+      hydrogen_quantity_wh: Number.parseFloat(cert.hydrogen_quantity_wh),
+      energy_consumed_wh: cert.energy_consumed_wh === null ? null : Number.parseFloat(cert.energy_consumed_wh),
+      embodied_co2: cert.embodied_co2 === null ? null : Number.parseFloat(cert.embodied_co2),
```

Comment:
Is this line designed to convert a string like **"1000"** to a number like **1000** ? What about if the string is **"1000.5"** ?

Reply:
good question, also wonder why not `parseInt()`

Reply:
So embodied_co2 should always be an int and we could use parseInt or parseFloat here. I went with parseFloat to reduce the risk of error in data manipulation as that would essentially DoS the service if there was bad data somehow in the db. 

Note the reason for parsing to a number at all is so it can be used as a number upstream. It just makes the JSON prettier than having a string.